app/controllers/holds_controller.rb

```diff
@@ -2,6 +2,8 @@
 
 class HoldsController < ApplicationController
   before_action :authenticate_user!
+  before_action :authorize_create!, only: :create
```

Comment:
```suggestion
  before_action :check_for_blanks!, only: :create
```

I think this wording would be clearer about what's going on, I think of authorization to be individual-based checking of permissions. I was going to suggest "validate_params" but it's all blank checking so, seemed better?

Reply:
changed 👍